.nvmrc

```diff
@@ -0,0 +1 @@
+v18.16.0
```

Comment:
can be used locally with `nvm install` otherwise it helps Netlify out https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript